packaging/python_wheels/mac/build.sh

```diff
@@ -0,0 +1,286 @@
+#!/bin/bash
+
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+
+function realpath
+{
+    python -c "import os; print(os.path.realpath('$1'))"
+}
+
+WORKSPACE=$(realpath `pwd`/../../../..)
+SOURCE_PATH=$WORKSPACE/Brion
+
+# Cannot use an earlier version because C++11 features don't compile otherwise
+export MACOSX_DEPLOYMENT_TARGET="10.9"
+export PYTHON_CONFIGURE_OPTS="--enable-universalsdk=/ --with-universal-archs=intel"
+export PYENV_ROOT=$WORKSPACE/pyenv
+export PATH="$PYENV_ROOT/bin:$PATH"
+export HTTP_PROXY=bbpproxy.epfl.ch:80
+export HTTPS_PROXY=bbpproxy.epfl.ch:80
+export http_proxy=bbpproxy.epfl.ch:80
+export https_proxy=bbpproxy.epfl.ch:80
+
+NUMPY_VERSION=1.12.0
+SPHINX_VERSION=1.3.6
+PYTHON_VERSIONS="2.7.14 3.4.7 3.5.4 3.6.4"
+
```

Comment:
all the other versions for boost, hdf5, etc. should also be grouped. and ideally shared with the linux build instructions.

Reply:
PYTHON_VERSIONS doesn't seem easy to unify at all. In the Linux script the variable contains a portion of paths inside the Docker container used to locate the different python versions. In fact, it's not really a Python version, but a CPython ABI version.